build/jenkins/Jenkinsfile.deployment

```diff
@@ -47,7 +46,6 @@ pipeline {
         }
```

Comment:
Doesn't look like this will run any thing

Reply:
I think this just might be github being weird. If you look at the whole file the changes seem to be there?